design/www/app/common/lastusestime.service.js

```diff
@@ -0,0 +1,45 @@
+ (function() {
+     'use strict';
+     angular.module('starter')
+         .factory('lastUsesTimeService', lastUsesTimeService);
+
+     function lastUsesTimeService(lastUsesTimeFactory, timeStorage, $interval, $localStorage) {
+         return {
+             updateTime: function(userId) {
```

Comment:
remove userId from this.
this function should always update time only for the current logged in person. 
always take user id from localStorage